main/vmaf/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "vmaf"
```

Comment:
this should probably be called libvmaf

Reply:
Even though it creates an executable file in /usr/bin/vmaf?

Reply:
hm probably not then, i thought vmaf was the whole repo while the libvmaf subdir was only a library

Reply:
but looking at it, in this case the program should be split off because it's pretty much the same size as the lib

so create a vmaf-libs subpackage containing self.default_libs